source/guides/views/custom-error-pages.md

```diff
@@ -4,20 +4,18 @@ title: Guides - Custom Error Pages
 
 # Custom Error Pages
 
-When a non successful request is returned, there are some special pages that an application presents to users.
-They have a generic graphic and some basic informations like the HTTP status code and the message.
+When an unsuccessful request is returned, there are some special pages that a Hanami application presents to users.
+These pages have a generic graphic and some basic information like the HTTP status code and the message.
 
-Hanami allows to customize them on an application basis.
-We need to create a template with the corresponding HTTP code as filename (eg. `apps/web/templates/500.html.erb`).
-
-Since now on, all the side errors will be presented using that template.
+Hanami allows us to customize them on a per-application basis.
+We just need to create a template with the corresponding HTTP code as the filename (e.g. `apps/web/templates/500.html.erb`).
+From then on, all the side errors will be presented using that template.
```

Comment:
Can you fix 'all the side errors' too? I'm not completely sure what that means. Instead maybe say something about how 500 is a system error (like an uncaught exception)?